### ISSUE TITLE
Python2 warning

### ISSUE THREAD

**Issue by MichaelClerx** [ModellingWebLab/cellmlmanip#15]
My default is still Python2, so just running `pytest` I get some unexpected errors:

```
/usr/lib/python2.7/site-packages/_pytest/python.py:395: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib/python2.7/site-packages/py/_path/local.py:662: in pyimport
    __import__(modname)
E     File "/home/michael/dev/weblab/cellmlmanip/tests/test_mathml2sympy.py", line 20
E   SyntaxError: Non-ASCII character '\xe2' in file /home/michael/dev/weblab/cellmlmanip/tests/test_mathml2sympy.py on line 20, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Could/should we add a check somewhere so that it tells you your python version (not these strings) are the issue?

**Comment by mirams** [comment]
I was reading something about Python2 going out of support in 2020. So perhaps we should make the WebLab completely Python 3?

**Comment by MichaelClerx** [comment]
We could maybe advertise only 3 but write for (and test on) both until something really ugly comes along. I'd like the underlying libraries to work on both at least

**Comment by tamuri** [comment]
Code fails with SymPy on Py2.7, essentially this:

```py
Python 2.7.14 |Anaconda, Inc.| (default, Oct  5 2017, 02:28:52)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from sympy import *
>>> Function(u'x’)  # <-- does not accept unicode strings
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tamuri/bin/anaconda3/envs/py27/lib/python2.7/site-packages/sympy/core/function.py", line 408, in __new__
    return UndefinedFunction(*args, **options)
  File "/Users/tamuri/bin/anaconda3/envs/py27/lib/python2.7/site-packages/sympy/core/function.py", line 782, in __new__
    ret = super(UndefinedFunction, mcl).__new__(mcl, name, bases, __dict__)
TypeError: type() argument 1 must be string, not unicode
```

**Comment by jonc125** [comment]
Given this is a pain to workaround, we'll only support Python 3 for now. We should fix #17 so this is clear to users, however.

**Comment by MichaelClerx** [comment]
I'm ok with going fully python3 officially and forever (not just for now). Will need a week to convert Myokit at some point this year, but that's long overdue anyway :-)
